recipes/opencolorio/all/conandata.yml

```
@@ -1,8 +1,16 @@
 sources:
+  "2.1.0":
+    url: "https://github.com/AcademySoftwareFoundation/OpenColorIO/archive/v2.1.0.tar.gz"
+    sha256: "81fc7853a490031632a69c73716bc6ac271b395e2ba0e2587af9995c2b0efb5f"
   "1.1.1":
     url: "https://github.com/AcademySoftwareFoundation/OpenColorIO/archive/v1.1.1.tar.gz"
     sha256: "c9b5b9def907e1dafb29e37336b702fff22cc6306d445a13b1621b8a754c14c8"
 patches:
+  "2.1.0":
+    - patch_file: "patches/2.1.0.patch"
```

can this have a more descriptive name please?

```suggestion
    - patch_file: "patches/fix-cmake-source-dir-and-targets.patch"
```